recipes/librmm/meta.yaml

```
@@ -0,0 +1,75 @@
+{% set name = "librmm" %}
+{% set version = "0.19.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  - url: https://github.com/rapidsai/rmm/archive/refs/tags/v{{ version }}.tar.gz
+    sha256: eddd5b35b016d665eb4e8ea2dd31497d913b3e1eb831124e4cb27cba747267a0
+    folder: rmm
+
+build:
+  number: 0
+  skip: true  # [(not linux)]
```

Should skip the CPU build, which is currently failing on CI

```suggestion
  skip: true  # [(not linux) or (cuda_compiler_version == "None")]
```

---

Thanks John, fixed in next commit.